### ISSUE TITLE
Bug: sudden 403 for certain models

### ISSUE THREAD

**Issue by johnsonps08** [tensorflow/hub#889]
### What happened?

Attempting to get following models:

- https://tfhub.dev/tensorflow/tfjs-model/blazeface
- https://tfhub.dev/mediapipe/tfjs-model/facemesh
- https://tfhub.dev/mediapipe/tfjs-model/iris

but getting 403 and CORS errors on GET.

### Relevant code - tested in Postman

```python
curl --location 'https://tfhub.dev/tensorflow/tfjs-model/blazeface/1/default/1/model.json?tfjs-format=file' \
--header 'authority: tfhub.dev' \
--header 'accept: */*' \
--header 'accept-language: en-US,en;q=0.9' \
--header 'cache-control: no-cache' \
--header 'origin: http://localhost:4200' \
--header 'pragma: no-cache' \
--header 'referer: http://localhost:4200/' \
--header 'sec-ch-ua: "Not.A/Brand";v="8", "Chromium";v="114", "Brave";v="114"' \
--header 'sec-ch-ua-mobile: ?0' \
--header 'sec-ch-ua-platform: "macOS"' \
--header 'sec-fetch-dest: empty' \
--header 'sec-fetch-mode: cors' \
--header 'sec-gpc: 1' \
--header 'user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36'
```

### Relevant log output

```shell
<h1>Forbidden</h1>
<h2>Error 403</h2>
```

### tensorflow_hub Version

0.12.0 (latest stable release)

### TensorFlow Version

2.8 (latest stable release)

### Other libraries

calls are going through these libraries:
`"@tensorflow-models/face-landmarks-detection": "0.0.3"
"@tensorflow/tfjs": "^3.15.0"`

### Python Version

3.x

### OS

macOS

**Comment by johnsonps08** [comment]
Additional info: this was working yesterday in both a local environment and a deployed environment. This isn't my project but you can go to [Webgazer.js collision demo](https://webgazer.cs.brown.edu/collision.html), inspect the page, and look at the network tab to see the exact same issue we're experiencing.

**Comment by akhorlin** [comment]
There was an intermittent issue. Please give it a try.

**Comment by johnsonps08** [comment]
@akhorlin thanks so much for the follow up. It is working now in all of our cases, local and deployed. I also noticed is is working on the demo for WebGazer.js.

Can you shed any light on what happened? Is the issue something that we could code more defensively for prevention of outage in the future?

**Comment by singhniraj08** [comment]
@johnsonps08,

There was an intermittent issue from our side which is fixed and there is nothing needed to be done from your end. You can refer [Common issues](https://www.tensorflow.org/hub/common_issues) in future for intermittent failures. Since this issue is fixed now, requesting you to close this issue. Thank you!

**Comment by johnsonps08** [comment]
Hi @singhniraj08 thanks for the update. Nothing in the Common Issues section you referenced seemed to match up with our issue. I'll be glad to mark this issue as closed with one last question: if we get 403 errors in the future, would it be best to open a ticket here right away or is there another, faster way to have that issue resolved?

**Comment by singhniraj08** [comment]
@johnsonps08,

We don't expect 403 errors, but since this occurred by some internal issues for which there is nothing needed to be done from your end, please feel free to create a new issue here if similar issue pops up in future. Thank you!

**Comment by gvgohil232** [comment]
Hello @singhniraj08 
Is there any changes going on in TensorFlow, I am getting a similar issue for fetch failed from 'https://tfhub.dev/tensorflow/tfjs-model/blazeface/1/default/1/model.json?tfjs-format=file') from origin 'http://localhost' has been blocked by CORS policy:

Please let me know how can I resolve this issue as my code is on live and it was working fine and now getting an error for CORS.

**Comment by HarshJain99** [comment]
I have also been facing the same issue for the blazeface model since this morning. Is this an intermittent issue or are any code updates expected from our end?



**Browser console log **

Access to fetch at 'https://www.kaggle.com/models/tensorflow/blazeface/frameworks/tfJs/variations/default/versions/1/model.json?tfjs-format=file&tfhub-redirect=true' (redirected from 'https://tfhub.dev/tensorflow/tfjs-model/blazeface/1/default/1/model.json?tfjs-format=file') from origin xxxxxxxxxxxxxxxxxxxxxxx has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

**Comment by vojdan** [comment]
I started getting this error a few hours ago:
```
Error: Request to https://tfhub.dev/mediapipe/tfjs-model/face_landmarks_detection/attention_mesh/1/model.json?tfjs-format=file failed with status code 403. Please verify this URL points to the model JSON of the model to load.
```

Official examples seem to have the same issue when switching the backend to `webgl`, but `cpu` seems to still work:
- https://storage.googleapis.com/tfjs-models/demos/face-detection/index.html?model=mediapipe_face_detector
- https://storage.googleapis.com/tfjs-models/demos/face-landmarks-detection/index.html?model=mediapipe_face_mesh

In the DevTools Network tab I get:


Request URL:
```
https://storage.googleapis.com/kagglesdsdata/models/1459/1725/model.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gcp-kaggle-com%40kaggle-161607.iam.gserviceaccount.com%2F20251018%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20251018T134133Z&X-Goog-Expires=259200&X-Goog-SignedHeaders=host&X-Goog-Signature=569a16d317d81afbda7df35eeab889618e1dac6ae8d42e918e3006505b51583d19aa56e174dbed4151b36505c5183363e7c4bbfd7e90ea46e8d1c3c571067550a45668a41d2017856b0c9ed146679dd2b32f8735504f851dea283d7a70fe88ba9a383db06d9da81942ce22a147b617e838e0b409d17a7cbea5177eab21e37a9de77131863484bac0a3190da80958358d6b0795a27d88be529c64c652f47fadf066c019c1c8bf13c57b3261e74c804aaeab05194081fb2069757928b2432d0f7693966128a00032d9ce11396bd1979e2db082165b2ee124486992e8419f670a15758a9c62ee717491568c64da298e80a71c0bffd1fe5cd93bc2ff97a2bdbaf3b9
```

Response:
```
<?xml version='1.0' encoding='UTF-8'?><Error><Code>SignatureDoesNotMatch</Code><Message>Access denied.</Message><Details>The request signature we calculated does not match the signature you provided. Check your Google secret key and signing method.</Details><StringToSign>GOOG4-RSA-SHA256
20251018T134133Z
20251018/auto/storage/goog4_request
3e732af564d2c97e573010fc5fb7450b85ed073365c2e4d52cd830accdb69900</StringToSign><CanonicalRequest>GET
/kagglesdsdata/models/1459/1725/model.json
X-Goog-Algorithm=GOOG4-RSA-SHA256&amp;X-Goog-Credential=gcp-kaggle-com%40kaggle-161607.iam.gserviceaccount.com%2F20251018%2Fauto%2Fstorage%2Fgoog4_request&amp;X-Goog-Date=20251018T134133Z&amp;X-Goog-Expires=259200&amp;X-Goog-SignedHeaders=host
host:storage.googleapis.com

host
UNSIGNED-PAYLOAD</CanonicalRequest></Error>
```

**Comment by leesohhyun** [comment]
I got the same error as above. It's also a 403. It was working fine until yesterday.
`
Error: Request to https://tfhub.dev/mediapipe/tfjs-model/face_detection/short/1/model.json?tfjs-format=file failed with status code 403. Please verify this URL points to the model JSON of the model to load.
`

**Comment by steam228** [comment]
Hello, i'm also getting a similar problem. 

in this case, directly from an example on the p5js web editor:

```
Error: Request to https://tfhub.dev/google/tfjs-model/movenet/multipose/lightning/1/model.json?tfjs-format=file failed with status code 403. Please verify this URL points to the model JSON of the model to load.
    at this.fetch (webpack:[//ml5/node_modules/@tensorflow/tfjs-core/dist/io/http.js:95:18](https://ml5/node_modules/@tensorflow/tfjs-core/dist/io/http.js:95:18))
```

**Comment by baig214** [comment]
I am also getting same issue as @vojdan

**Comment by arjun42squar** [comment]
I’m trying to load the TensorFlow Universal Sentence Encoder model in a browser app using @tensorflow-models/universal-sentence-encoder, but the request to the model URL returns 403 Forbidden.

`"Request to https://tfhub.dev/tensorflow/tfjs-model/universal-sentence-encoder-lite/1/default/1/model.json?tfjs-format=file failed with status code 403. Please verify this URL points to the model JSON of the model to load."`

Expected:
Model loads successfully as before.

Actual:
Fetch fails with 403 error from both tfhub.dev and storage.googleapis.com URLs.

Notes:

This worked earlier but started failing recently.

**Comment by juliahansmeyer** [comment]
I am also getting the same error when trying to load the google Yamnet model.

**Comment by kempermo** [comment]
Same over here. Getting 403 error when trying to access https://tfhub.dev/google/tfjs-model/movenet/multipose/lightning/1/model.json?tfjs-format=file Was there a change in authentication? Would be nice to get it fixed.

**Comment by misbahkun** [comment]
halo! i also got eror similiar, can help sir 
Request to https://tfhub.dev/tensorflow/tfjs-model/blazeface/1/default/1/model.json?tfjs-format=file failed with status code 403. Please verify this URL points to the model JSON of the model to load.

**Comment by KeijiBranshi** [comment]
Hi all. Thanks for raising the issue. We're looking into it, and will post updates to the thread here: https://github.com/tensorflow/hub/issues/931

**Comment by Arjun812** [comment]
The URL is returning a 403 error. Rest assured, this is not a permanent issue. The cause is an external outage as TensorFlow is currently performing system maintenance. They anticipate fixing the problem and restoring service within 24 hours.
